app/common.go

```diff
@@ -3,6 +3,7 @@ package app
 import (
 	"time"
 
+	mltrainingpb "go.viam.com/api/app/mltraining/v1"
```

Comment:
(q) I'm curious about this. If we're importing proto types specifically for `mltraining` and made the flyby of moving conversion functions _out of_ separate files into the relevant client file, then why are we importing this in `common.go` and defining mltraining conversion functions here as opposed to in the `mltraining_client.go` file? Can the constants that use them not just be defined in the `test` file where they're used?

edit: ahh I see that these are used (in a roundabout way) by app client. okay, disregard!

Reply:
Actually, now that I think about it, it's a little strange that we're defining constants used only for tests (e.g., `locationID`) and constants used in actual business logic (e.g., `UploadChunkSize`) in the same file. This looks like it's mostly test constants (`UploadChunkSize` is the only one I see from a quick glance that I think is used in business logic), but the file doesn't make that clear at all. And below, we define types that are actually relevant in multiple places.

Probably not appropriate for this PR, but we should probably create a ticket for a quick refactor here to make it clear what's only for testing and what is actual business logic shared by multiple clients (just splitting this up into two files seems easiest imo).

Reply:
This makes sense to me, I can easily do it in this ticket.

Reply:
I split it up into `common.go` and `testutils.go`. Let me know if I should change the names!